From: "Gary V. Vaughan" <gary@gnu.org>
Date: Thu, 12 Dec 2013 09:15:12 +1300
Subject: doc: fix LDoc manual page custom_see_handler pattern.

* ext/posix/config.ld (custom_see_handler): Add a missing closing
`%)`.
(upat): Pull the format string up per LDoc documentation.

Signed-off-by: Gary V. Vaughan <gary@gnu.org>
---
 ext/posix/config.ld | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/ext/posix/config.ld b/ext/posix/config.ld
index c8994ac..2ca615f 100644
--- a/ext/posix/config.ld
+++ b/ext/posix/config.ld
@@ -7,9 +7,10 @@ tparam_alias 'string'
 tparam_alias 'int'
 tparam_alias 'bool'
 
-custom_see_handler('^([%w_]+)%((%d)$',function(name,section)
-    local url = ("http://www.kernel.org/doc/man-pages/online/pages/man%s/%s.%s.html"):format(section,name,section)
-    local name = name .. '(' ..section..')'
+local upat = "http://www.kernel.org/doc/man-pages/online/pages/man%s/%s.%s.html"
+custom_see_handler('^([%w_]+)%((%d)%)$',function(name,section)
+    local url = upat:format(section,name,section)
+    local name = name .. '(' .. section .. ')'
     return name, url
 end)
 
